index.html

```diff
@@ -87,10 +87,97 @@ <h1 class="text-3xl pb-4">Colors</h1>
           </div>
         </div>
 
-        <div class="flex-col flex-shrink-0 px-4 py-4">
+        <div class="flex-col flex-shrink-0 px-4 py-4" style="font-family: Overpass;">
           <h1 class="text-3xl pb-4">Typography</h1>
           <div class="container">
-            <p>HELLO I AM HERE.</p>
+            <h2 class="text-2xl pb-4">This is Overpass</h2>
+            <p>Inspired by the typeface used in road signage all around America, Overpass represents the UX Wizards’ ever expanding reach. 
+            </p><br>

```

Comment:
This `<br>` seems odd since it's the only one you use in the whole section, but it's fine for now.